CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-python/compare/v1.5.0-0.24b0...HEAD)
 
+- Add metrics API
+  ([#1887](https://github.com/open-telemetry/opentelemetry-python/pull/1887))
 - `opentelemetry-semantic-conventions` Update to semantic conventions v1.6.1
```

Comment:
CHANGELOG?

Reply:
Ok. This CHANGELOG is a little funny because these changes will not be release in v1.5.0. Should probably split it into a separate file until we merge it.